x-pack/legacy/plugins/maps/common/descriptor_types.d.ts

```diff
@@ -18,3 +18,30 @@ export interface ILayerDescriptor {
   id: string;
   label?: string;
 }
+
+interface IFieldMetaDescriptor {
```

Comment:
I don't think this interface is needed at all. Why do IRangeFieldMeta and ICategoryFieldMeta need to inherit from anything? They are just plain old objects.